README.md

```diff
@@ -327,46 +327,36 @@ Use of this source code is governed by a MIT license that can be found in the [L
 
 ## External sources
 
-In `rv32emu` repository, there are some prebuilt ELF files for testing purpose.
-* `aes.elf` : See [tests/aes.c](tests/aes.c)
-* `captcha.elf` : See [tests/captcha.c](tests/captcha.c)
+There are some prebuilt executables in [rv32emu-prebuilt](https://github.com/sysprog21/rv32emu-prebuilt) for testing and benchmarking purpose.
+The prebuilt executables are built from GNU/GCC v9 and [xPack GCC v14.2.0-1](https://github.com/xpack-dev-tools/gcc-xpack) under Ubuntu-20.04.
+
+To fetch the prebuilt executables, run the following command:
+
+```shell
+$ make build-artifact
+```
+
+Or, to compile the executables from the source, run:
+
+```shell
+$ make build-artifact USE_PREBUILT=0
+```
+
+Notice that the RISC-V cross-compiler is required to build the binary.
+
+In default, `rv32emu` will download the tarball from GitHub and extract the executables into `build/`.
+
+There are still some prebuilt ELF files for testing purpose under `build/`.
```

Comment:
Create a file `docs/prebuilt.mk` detailing the prebuilt ELF files with a comprehensive listing about their origins.